email.go

```diff
@@ -15,7 +15,7 @@ type Email struct {
 // Options encapsulate headers the client can pass in requests to rspamd.
 type Options struct {
 	flag   int
-	weight float64
+	weight int32
```

Comment:
If it's parsed as long wouldn't `int64` be more appropriate? Not that it matters much since we aren't sending any high values anyway

Reply:
A long in C is _at least_ 32 bits. So I decided to go with the lower bound.